.github/workflows/update_collector_and_scanner.yaml

```diff
@@ -0,0 +1,95 @@
+name: Update collector and scanner version
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: 0 5 * * 1
+jobs:
+  update-collector:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        ref: master # this is our target branch
+        fetch-depth: 0 # we need to fetch all branches
+    - name: Checkout collector repo
+      uses: actions/checkout@v3
+      with:
+        repository: stackrox/collector
+        path: deps/collector
+        fetch-depth: 0 # we need to fetch tags
+    - name: Update version
+      run: make -sC deps/collector tag | tee COLLECTOR_VERSION
+    - name: Create Pull Request
+      id: cpr
+      uses: peter-evans/create-pull-request@v4
+      with:
+        token: '${{ secrets.ROBOT_ROX_GITHUB_TOKEN }}'
+        commit-message: Update COLLECTOR_VERSION
+        committer: RoxBot <roxbot@stackrox.com>
+        author: RoxBot <roxbot@stackrox.com>
+        branch: update_collector_version
+        signoff: false
+        delete-branch: true
+        title: Update COLLECTOR_VERSION
+        body: |
+          Update COLLECTOR_VERSION to latest master version
+        labels: |
+          ci-all-qa-tests
+          dependencies
+        assignees: janisz
```

Comment:
Do we need an assignee? I think these PRs will be mostly on the collector team to review and get merged.